lisa/transformers/kernel_installer.py

```diff
@@ -212,7 +212,7 @@ def install(self) -> str:
             f"deb {self.repo_url} {version_name} "
             f"restricted main multiverse universe"
         )
-        ubuntu.wait_running_package_process()
+        ubuntu._initialize_package_installation()
```

Comment:
1. This part we need to think about how to refactor the code. Not to call a private method (starts with `_`) out of the class.
2. The original method name makes sense "wait running package process", because it needs to add repo. But it doesn't make sense to "initialize package installation", because no package will be installed after this line immediately.

Reply:
updated